### DISCUSSION TITLE
Preferred Way of Filtering Exceptions?

### DISCUSSION THREAD

**Discussion by Mike-E-angelo** [datalust/seq-tickets#1666] [Help] [answered]
I am seeing certain exceptions in my logs that are [known issues](https://github.com/EdCharbeneau/BlazorSize/issues/82) and would like to filter them out.  What is the recommended way of doing so?  My guess/hunch is to make a new signal called `Ignored Exceptions` or some equivalent and enable it, but wanted to see if there's a more fancy/preferred way of doing this before proceeding.

I am also interested in filtering these for email notifications as well.

Thank you for any assistance.

**Comment by liammclennan** [comment, marked as answer]
Hi Mike,

Your idea is a good one, or, if you don't want to see those events at all, you could add a filter on the API key.

**Reply by Mike-E-angelo** [reply]
Thank you very much for the confirmation @liammclennan 🙏
